pyodide_build/vendor/loky.py

```diff
@@ -170,7 +170,10 @@ def _count_physical_cores():
     try:
         if sys.platform == "linux":
             cpu_info = subprocess.run(
-                "lscpu --parse=core".split(), capture_output=True, text=True
+                "lscpu --parse=core".split(),
+                capture_output=True,
+                text=True,
+                check=False,
```

Comment:
Commands like these are quite unlikely to fail anywhere, but perhaps `check=True` is a better option.

```suggestion
                "lscpu --parse=core".split(),
                capture_output=True,
                text=True,
                check=True,
```